v1.0/Specification.md

```diff
@@ -332,6 +332,7 @@ Table with terms used in the specification.
 | Term | Description |
 | --- | --- |
 | **Archival Information Package (AIP)** | *From OAIS:* An Information Package, consisting of the Content Information and the associated Preservation Description Information (PDI), which is preserved within an OAIS. |
+| **Artefact** | An artefact is a data object in a spreadsheet which is not a significant property and hence not important for future reuse of the spreadsheet, but removing the object may not be considered neither necessary nor worthwhile. Therefore, the data object may remain in the spreadsheet and considered unnecessary for rendering or future migration. |
```

Comment:
Is it always a data object, and is that a defined term? There may be something better/less vanilla in the ODF spec terminology.

Reply:
You are right. I checked the ODF spec and they consistently use the term "object" and not "data object". They specify types such as:
- Graphical/graphic object
- Drawing object
- OLE object
- Presentation object
- Embedded object
- Database object
- 3D object
- Extrusion object
- Projected object
- Query object
- Math object
- Spreadsheet object
- Layout object

I think denoting the term to "object" without any prefix is the best way forward.

Reply:
And yet you use it in the last sentence ;)